firebase_admin/db.py

```diff
@@ -125,31 +125,70 @@ def child(self, path):
         full_path = self._pathurl + '/' + path
         return Reference(client=self._client, path=full_path)
 
-    def get(self):
-        """Returns the value at the current location of the database.
+    def etag(self):
```

Comment:
Lets remove this method. Our API review team decided to implement `get_if_changed()` instead of `etag()`.